metrics/web/lib/project_groups/presentation/widgets/project_checkbox_list.dart

```diff
@@ -27,8 +32,11 @@ class ProjectCheckboxList extends StatelessWidget {
         }
 
         if (projectCheckboxViewModels.isEmpty) {
```

Comment:
It seems like we should display the `no search results` text only when we have a project name filter, and there are no projects matches such filter.